data/channels.csv

```diff
@@ -21513,145 +21513,142 @@ NST.ru,NST,НСТ;Настоящее Страшное Телевидение,,VG
+NTV.bd,NTV,এনটিভি,,International Television Channel Ltd,BD,,Dhaka,c/BD,ben,general,FALSE,2003-07-02,,,https://www.ntvbd.com/,https://i.imgur.com/l75bDTx.png
 NTV.br,NTV,Nossa TV,,,BR,BR-MG,Patos de Minas,s/BR-MG,por,general;entertainment,FALSE,,,,https://www.ntv.tv.br/,https://i.imgur.com/n79nkVC.png
-NTV.ca,NTV,,,,CA,,Newfoundland,c/CA,eng,,FALSE,,,,,https://i.imgur.com/KQtQg5y.png
-NTV.ci,NTV,,,,CI,,,c/CI,fra,general,FALSE,,,,http://www.notretelevision.org/,https://i.imgur.com/Lj4WvXZ.png
-NTV.cl,NTV,,,,CL,,,c/CL,spa,culture;education,FALSE,2021-08-08,,,https://www.tvn.cl/ntv,https://i.imgur.com/NnkSyta.png
-ntv.de,n-tv,,,,DE,,,c/DE,deu,,FALSE,,,,https://www.n-tv.de/,https://i.imgur.com/tUSXrVm.png
+NTV.ca,NTV,CJON-DT,,Stirling Communications International,CA,CA-NL,St. John's,c/CA,eng,general,FALSE,1955-09-06,,,https://ntv.ca/,https://i.imgur.com/KQtQg5y.png
+NTV.ci,NTV,,,Our TV,CI,,Abidjan,c/CI,fra,general,FALSE,2018-10-12,,,http://www.notretelevision.org/,https://i.imgur.com/Lj4WvXZ.png
+NTV.cl,NTV,,,State of Chile,CL,,Santiago,c/CL,spa,culture;education,FALSE,2021-08-08,,,https://www.tvn.cl/ntv,https://i.imgur.com/NnkSyta.png
 NTV.gh,NTV,,,,GH,,,c/GH,eng,,FALSE,,,,https://www.ntvghana.com/,https://i.imgur.com/pYNj2jj.png
-NTV.kh,NTV,,,,KH,,,c/KH,khm,,FALSE,,,,,https://i.imgur.com/SoBhuIb.png
-NTV.mn,NTV,,,,MN,,,c/MN,mon,,FALSE,,,,http://ntv.mn/,https://i.imgur.com/2JbqbPY.png
-NTV.mv,NTV,,,,MV,,,c/MV,div,general,FALSE,2023-09-07,,,,https://i.imgur.com/jnXpcwV.png
-NTV.na,NTV,Network Television,,,NA,,,c/NA,eng,general,FALSE,2022-02-08,,,https://oneuptwo.com/,https://i.imgur.com/c7jUmGS.jpg
-NTV.rs,NTV,НТВ,,Slađana Ostojić;Narodne novine doo Niš,RS,,,c/RS,eng,,FALSE,,,,,https://i.imgur.com/xNf4l8u.png
-NTV.ru,NTV,НТВ,,Gazprom Media,RU,,,c/RU,rus,general,FALSE,1967-11-04,,,https://www.ntv.ru/,https://i.imgur.com/9r1BKzK.png
-NTV.tr,NTV,,,,TR,,,c/TR,tur,,FALSE,,,,https://www.ntv.com.tr/,https://i.imgur.com/jXbs8FZ.png
-NTV7.my,NTV7,,,Media Prima;Ministry of Education,MY,,,c/MY,zho,,FALSE,,,,https://www.xtra.com.my/,https://i.imgur.com/jDWR3xI.png
-NTVAmerica.us,NTV-America,НТВ-Америка,,,US,,,c/US,rus,,FALSE,2002-09-25,,,http://ntva.tv/,https://upload.wikimedia.org/wikipedia/commons/thumb/e/ef/NTV-America.jpg/512px-NTV-America.jpg
-NTVBaltic.ru,NTV-Baltic,НТВ-Baltic,,,RU,,,c/RU,rus,,FALSE,,,,http://www.ntvmir.ntv.ru/,https://i.imgur.com/vfCjWyh.png
-NTVBelarus.by,NTV-Belarus,НТВ-Беларусь,,,BY,,,c/BY,bel,general,FALSE,2006-07-04,,,http://tvr.by/rus/ntv.asp,https://i.imgur.com/QLuUIeB.png
-NTVEurope.bd,NTV Europe,,,,BD,,,c/UK,ben,general,FALSE,,,,https://europentv.ntvbd.com/,https://europentv.ntvbd.com/sites/all/themes/sloth/logo.png
-ntvEvent.de,n-tv Event,,,,DE,,,c/DE,deu,,FALSE,,,,,https://i.imgur.com/9FPXcFY.jpg
-ntvHD.at,n-tv HD,,,RTL Group,AT,,,c/AT,deu,,FALSE,,,,https://www.n-tv.de/,https://upload.wikimedia.org/wikipedia/commons/thumb/f/fd/N-tv-HD-September.svg/512px-N-tv-HD-September.svg.png

```

Comment:
Why delete `ntvHD.at`?

Reply:
Please read the Wikipedia article for clarification; also, please approve.